browser/extensions/api/ftx_api.cc

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/extensions/api/ftx_api.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "brave/browser/ftx/ftx_service_factory.h"
+#include "brave/browser/profiles/profile_util.h"
+#include "brave/common/extensions/api/ftx.h"
+#include "brave/components/ftx/browser/ftx_service.h"
+#include "brave/components/ftx/browser/regions.h"
+#include "brave/components/ftx/common/pref_names.h"
+#include "brave/components/ntp_widget_utils/browser/ntp_widget_utils_region.h"
+#include "chrome/browser/extensions/api/tabs/tabs_constants.h"
+#include "chrome/browser/extensions/extension_tab_util.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/prefs/pref_service.h"
+#include "extensions/browser/extension_util.h"
+#include "extensions/common/constants.h"
+
+namespace {
+
+FTXService* GetFTXService(content::BrowserContext* context) {
+  return FTXServiceFactory::GetInstance()->GetForProfile(
+      Profile::FromBrowserContext(context));
+}
+
+bool IsFTXAPIAvailable(content::BrowserContext* context) {
```

Comment:
I think IsFTXAvailable should be defined somewhere else so the implementation stays in sync with https://github.com/brave/brave-core/pull/8629/files#diff-05d82cf839b090feae4f8623d86beebe9c71b528eca857c3a235c19d65a7fbfeR21

Reply:
actually there's really not much reason for this because `GetFTXService` will return null in that case

Reply:
removed via https://github.com/brave/brave-core/compare/6f4c503619e82a440ef7db9782288e922c37d5e4..f4d887c0baf39543331107f2e22f9b0ad4986c02